src/components/Value/index.tsx

```diff
@@ -28,10 +28,10 @@ export const ValueSection = ({ values, ...sectionProps }: IValueSection) => {
       <Container title="Our Values" className="border-b px-0 pb-20">
         <div className="container flex flex-no-wrap justify-center md:justify-around md:px-0 sm:flex-wrap">
           {values.map((value, key) => (
-            <div className="flex flex-col w-100 sm:w-100 sm:items-center sm:text-center pr-16 sm:pr-0">
+            <div className="flex flex-col text-center w-100 sm:w-100 sm:items-center sm:text-center pr-16 sm:pr-0">
```

Comment:
Does `w-100` do anything?

```suggestion
            <div className="flex flex-col text-center sm:items-center sm:text-center pr-16 sm:pr-0">
```